tests/DependencyInjection/Compiler/ExtensionCompilerPassTest.php

```diff
@@ -545,17 +543,6 @@ private function getContainer(): ContainerBuilder
             ->register('file_locator')
             ->setClass(FileLocatorInterface::class);
 
-        $blockExtension = new SonataBlockExtension();
-        /*
-         * TODO: remove "http_cache" parameter when support for SonataBlockBundle 4 is dropped.
```

Comment:
I'm not sure to understand the comment @dmaicher, shouldn't we keep
```
$blockExtension->load(['sonata_block' => []], $container);
```
or it's a useless call ?

Reply:
well tests are passing :smile: So I think it was a useless call. No need to load the extension. This was just loaded to prevent a deprecation I believe.